migrations/migrations/R__Create-Test-Data.sql

```diff
@@ -1850,6 +1876,32 @@ VALUES
   ) ON CONFLICT
 DO NOTHING;
 
+INSERT INTO
+  public.officer (
+    officer_guid,
+    user_id,
+    create_user_id,
+    create_utc_timestamp,
+    update_user_id,
+    update_utc_timestamp,
+    person_guid,
+    office_guid,
+    auth_user_guid
+  )
+VALUES
+  (
+    'b17ee2c1-a26b-4911-ac6f-810b8fdfaab3'::uuid,
+    'JFUNK',
+    'FLYWAY',
+    '2023-06-29 22:16:16.754',
+    'FLYWAY',
+    '2024-01-22 22:20:48.186',
+    '97f3cee5-6f4a-410f-810f-d431595fccee'::uuid,
+    '4a5a94b1-bd47-4611-a577-861d97089903'::uuid,
```

Comment:
This (office_guid) should be c3d8519c-73cb-48a1-8058-358883d5ef4f for COSHQ otherwise you will show up on pilot complaints as an assignee.   

Alternative (as of release Orca) you can set this value to null.